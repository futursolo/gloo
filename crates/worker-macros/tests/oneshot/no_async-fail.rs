#[gloo::worker::oneshot]
fn Worker(input: u32) -> u32 {
    input
}

fn main() {}
