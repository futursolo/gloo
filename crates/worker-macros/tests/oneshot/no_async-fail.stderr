error: oneshot functions must be async
 --> tests/oneshot/no_async-fail.rs:1:1
  |
1 | #[gloo::worker::oneshot]
  | ^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `gloo::worker::oneshot` (in Nightly builds, run with -Z macro-backtrace for more info)
